pages/githubandmarkdown.md

```diff
@@ -87,6 +87,9 @@ Then, you can create a pull request by clicking the `Pull requests` button and t
 
 After the pull request is completed, you'll be able to see this on open-learning-exchange.github.io in addition to your personal page. Let us know when you have completed this step in the [gitter.im chat](https://gitter.im/open-learning-exchange/chat).
 
+**NOTE**: Try to add and experiment with as many markdown elments as you can and make your page attractive. A list of sample profile pages can be found [**here**](https://github.com/open-learning-exchange/open-learning-exchange.github.io/tree/master/pages/profiles). Be creative.  
```

Comment:
@LKhadka It looks good on rawgit but there is a  typo "elments". please correct it thanks

Things I've reviewed:

[x] Check for issue number in pull request title
[x] Are there any unneeded files in the pull request?
[x] Did they make a branch for their patch?
[x] Does the pull request actually fix the issue?
[x] Check the pull request on rawgit, does it display without any errors?
[x] Is there any merge conflicts?
[x] Make sure that people use their GitHub accounts when making commits through git